doc/OnlineDocs/contributed_packages/mindtpy.rst

```diff
@@ -6,16 +6,52 @@ allows users to solve Mixed-Integer Nonlinear Programs (MINLP) using decompositi
 These decomposition algorithms usually rely on the solution of Mixed-Intger Linear Programs
 (MILP) and Nonlinear Programs (NLP).
 
-MindtPy currently implements the Outer Approximation (OA) algorithm originally described in
-[`Duran & Grossmann, 1986`_] and the Extended Cutting Plane (ECP) algorithm originally described in [`Westerlund & Petterson, 1995`_]. Usage and implementation
-details for MindtPy can be found in the PSE 2018 paper Bernal et al.,
+The following algorithms are currently available in MindtPy:
+
+- **Outer Approximation (OA)** [`Duran & Grossmann, 1986`_]
+- **LP/NLP based branch and bound (LP/NLP)** [`Quesada & Grossmann 1992`_]
+- **Extended Cutting Plane (ECP)**  [`Westerlund & Petterson, 1995`_]
+- **Global Outer Approximation (GOA)**
```

Comment:
Add Kesavan reference here (Kesavan, Padmanaban, et al. "Outer approximation algorithms for separable nonconvex mixed-integer nonlinear programs." Mathematical Programming 100.3 (2004): 517-535.) and MC++ reference (https://omega-icl.github.io/mcpp/)